src/endpoints/catalog.js

```diff
@@ -273,14 +273,15 @@ class ShopperCatalogEndpoint extends ShopperCatalogQuery {
   constructor(endpoint) {
     super()
     const config = { ...endpoint } // Need to clone config so it is only updated in PCM
+    config.version = ''
```

Comment:
instead of setting the config version to an empty string here could you set it to undefined during the spread.

```
const config = { ...endpoint, version: undefined } // Need to clone config so it is only updated in PCM
```
then I'll get this merged. Thanks

Reply:
My bad. Will add it and update PR. Thanks